.travis.yml

```diff
@@ -3,34 +3,33 @@ language: cpp
 os: linux
 dist: bionic
 
-before_install:
-  - sudo apt-get -y install libxml2-dev
-
 jobs:
   include:
 
     ################
     # Clang on linux
 
-    #    # Clang legacy
-    #    - name: Clang-9 Debug
-    #      stage: Legacy
-    #      env: CXX=clang++-9 BUILD_TYPE=Debug LLVM_VERSION=9
-    #      addons: &clang9
-    #        apt:
-    #          sources:
-    #            - sourceline: 'deb http://apt.llvm.org/bionic/ llvm-toolchain-bionic-9 main'
-    #              key_url: https://apt.llvm.org/llvm-snapshot.gpg.key
-    #            - sourceline: 'deb http://apt.llvm.org/bionic/ llvm-toolchain-bionic-10 main'
-    #              key_url: https://apt.llvm.org/llvm-snapshot.gpg.key
-    #          packages:
-    #            - clang-9
-    #            - llvm-9-dev
-    #            - clang-tidy-10
-    #    - name: Clang-9 Release
-    #      stage: Legacy
-    #      env: CXX=clang++-9 BUILD_TYPE=Release LLVM_VERSION=9
-    #      addons: *clang9
+    # Clang legacy
+    - name: Clang-9 Debug
+      stage: Legacy
+      env: CXX=clang++-9 BUILD_TYPE=Debug LLVM_VERSION=9
+      addons: &clang9
+        apt:
+          sources:
+            - sourceline: 'deb http://apt.llvm.org/bionic/ llvm-toolchain-bionic-9 main'
+              key_url: https://apt.llvm.org/llvm-snapshot.gpg.key
+            - sourceline: 'deb http://apt.llvm.org/bionic/ llvm-toolchain-bionic-10 main'
+              key_url: https://apt.llvm.org/llvm-snapshot.gpg.key
+          packages:
+            - g++-9
+            - clang-9
+            - llvm-9-dev
+            - clang-tidy-10
+            - libxml2-dev
```

Comment:
Small nit: Should we put general deps into the `before_install`? This makes it easier to add new builds to the matrix.